### PR TITLE
[DOCS] Fixed KLDiv example

### PR DESCRIPTION
**Pull request by elisim**
Small import fix to make the example run

### CODE REVIEW

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -449,6 +449,8 @@ class KLDivLoss(_Loss):
     Examples::
 
         >>> import torch.nn.functional as F
+        >>> from torch import nn
+        >>> import torch
```
Comment (vadimkantorov) [inline torch/nn/modules/loss.py:453]: should maybe `import torch` be moved above `import torch.nn.functional as F`? also, style of `from torch import nn` does not match the style of `import torch.nn.functional as F`

I would propose (or at least make them the same style):
```
import torch
import torch.nn as nn
import torch.nn.functional as F
```

Comment (elisim) [reply torch/nn/modules/loss.py:453]: sure

**Review by albanD — CHANGES_REQUESTED** [review]
Hey!
Actually you should even remove the functional import from here as these are expected to always be already done in all our docstrings as you can see in the doctest config here:
https://github.com/pytorch/pytorch/blob/082251e76b93b277ff2791d0e2b64934add34644/test/run_test.py#L912-L914

**Review by albanD — APPROVED** [review]
Thanks!
